hr_timesheet_sheet/__manifest__.py

```diff
@@ -25,6 +25,11 @@
         "views/hr_employee_views.xml",
         "views/account_analytic_line_views.xml",
         "views/res_config_settings_views.xml",
-        "templates/assets.xml",
     ],
+    'assets': {
+        'web.assets_backend': [
+            '/hr_timesheet_sheet/static/src/js/backend.js',
+            '/hr_timesheet_sheet/static/src/scss/backend.scss'
```

Comment:
Might be an issue after all. 
```suggestion
            'hr_timesheet_sheet/static/src/js/backend.js',
            'hr_timesheet_sheet/static/src/scss/backend.scss'
```